jobs/build/ocp/Jenkinsfile

```diff
@@ -317,7 +318,7 @@ node(TARGET_NODE) {
                                 grunt build
 
                                 git add dist
-                                git commit -m "Merge master into enterprise-${BUILD_VERSION}" --allow-empty
+                                git commit -m "Merge master into enterprise-${BUILD_VERSION}\nWeb console commit: ${WEB_CONSOLE_COMMIT}" --allow-empty
                             """
```

Comment:
i dont think this is the right commit, if i understand this right this is the commit that is going to the enterprise branch of the web console repo.  What we need updated is the commit that gets generated from hack/vendor-console.sh that commits the bindata.go file into ose

Reply:
Ah! That explains some things. I'll update to call that instead.